client/js/jsonedit.js

```diff
@@ -267,6 +267,19 @@ const jeCommands = [
       }
     }
   },
```

Comment:
I thought maybe this could be deleted, but then you don't get the upload button if the image is something like: i/game-pieces/2D/Checkers-2D.svg.  But leaving this in causes duplicate buttons to appear if you select the /assets/ text instead of just putting the cursor there.  This doesn't really bother me, so I'm going to approve.  But just pointing it out.

![Editor](https://github.com/ArnoldSmith86/virtualtabletop/assets/76912527/f90be789-9cfc-41e8-9043-d0e3ae866d65)

Reply:
Didn't think about the duplicates. Will fix.